server/routerlicious/packages/services/src/redisThrottleStorageManager.ts

```diff
@@ -31,6 +32,7 @@ export class RedisThrottleStorageManager implements IThrottleStorageManager {
 
         client.on("error", (error) => {
             winston.error("Throttle Manager Redis Error:", error);
+            Lumberjack.error("Throttle Manager Redis Error:", error);
```

Comment:
Same thing: `Lumberjack.error`'s second positional parameter is a properties parameter. Error should be the third parameter, so the above might not produce the desired effect. Can you do instead:
```typescript
Lumberjack.error("Throttle Manager Redis Error:", undefined, err);
```
https://github.com/microsoft/FluidFramework/blob/4202d915b7f92f9afeb607219cb5369b362905a1/server/routerlicious/packages/services-telemetry/src/lumberjack.ts#L90